.travis.yml

```diff
@@ -1,30 +1,51 @@
 language: node_js
-
 sudo: required
 
-node_js:
- - "6"
```

Comment:
Nodejs v6 removal is scheduled for May (see https://github.com/telefonicaid/lightweightm2m-iotagent/issues/140). Two alternatives:

1. Include the corresponding Nodev6 section
2. Wait to May 2019 to process this PR

In my opinion it should be better option 1.

Reply:
Fixed  85b9ce4e - I thought it was *March*